lte/gateway/deploy/agw_install.sh

```diff
@@ -86,6 +110,22 @@ WantedBy=multi-user.target" > $AGW_INSTALL_CONFIG
   reboot
 fi
 
+echo "Checking if kernel headers was installed, else install"
+if ! [ "$(dpkg -l  | grep headers-4.9.0-9-amd64)" ]; then
+  # Get from alternative repository and add as debian packages local repo
+  wget http://cdimage.debian.org/mirror/cdimage/archive/9.9.0/amd64/iso-cd/debian-9.9.0-amd64-xfce-CD-1.iso
+  mkdir -p /mnt/disk
+  mount -o loop debian-9.9.0-amd64-xfce-CD-1.iso /mnt/disk
+  echo 'deb file:///mnt/disk/ stretch main contrib' >> /etc/apt/sources.list
+  sudo apt-get update --allow-unauthenticated
+  # Install kernel headers
+  sudo apt -y install linux-headers-4.9.0-9-amd64 --allow-unauthenticated
+  # Clean headers files and umount
+  umount /mnt/disk
+  rm debian-9.9.0-amd64-xfce-CD-1.iso
+  sed -i '/deb file:\/\/\/mnt\/disk\//d' /etc/apt/sources.list
+fi
+  
```

Comment:
Why does this code block appear twice in the script? Seems like we should only have it in one place, but if we need it in both then you should pull it into a function

Reply:
Hi @xjtian , that was an error, removed the duplicated code.